Makefile

```diff
@@ -85,10 +85,12 @@ updater: config-updater package
 verify-makensis:
 ifneq "$(MAKENSIS_VERSION)" "v08-Feb-2016.cvs"
 ifneq "$(MAKENSIS_VERSION)" "v12-Dec-2016.cvs"
-	echo "$(RED)Please upgrade NSIS installer requires >= v2.5.0.";
-	echo "> If new version installed, Makefile needs updating (verify-makensis)";
-	echo "> brew update; brew upgrade makensis$(RESET)";
-	exit 1;
+ifneq "$(MAKENSIS_VERSION)" "v25-Jul-2017.cvs"
```

Comment:
If we're targeting specific versions here, we should clearly document that.

If we're mainly interested in specifying a minimum version, I'd prefer we go ahead and make the change to do a greater than or equal to comparison.